modules/ROOT/pages/map-users-roles-ldap-task.adoc

```diff
@@ -1,39 +1,152 @@
-= To Map Users to Roles in an LDAP Group
+= Map Single Sign-On Users to Roles
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can map users in a federated organization's LDAP group to an Anypoint role. Your Anypoint Platform organization needs to use an external identity provider, such as PingFederate.
+You can map users in a federated organization's group to an Anypoint role. Your Anypoint Platform organization must use an external identity provider, such as PingFederate.
 
-After you perform this mapping, users in an organization can sign into Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using LDAP. +
+After you perform this mapping, users in an organization can sign in to Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using SAML, OpenID Connect (OIDC), or LDAP. +
 This ensures security of credentials and maintains organizational roles for accessing privileged information.
```

Comment:
```suggestion
This helps to ensure secure credentials and to maintain organizational roles for accessing privileged information.
```
Because of Legal issues, we can never say that anything is ensured by our products, so...helps to ensure, per Style Ref.